### PR TITLE
kiln: recursive `|rm`

### PR DESCRIPTION
**Pull request by ryjm**
Increments the commit number every time, is there a better way to do this?

image

### CODE REVIEW

**Comment by ryjm** [comment]
ah looks like `furl` might be what i'm looking for.

File: pkg/arvo/lib/hood/kiln.hoon [hunk 1]
```diff
@@ -1256,11 +1256,18 @@
 ::
 ++  poke-rm
   |=  a=path
+  =|  c=(list (unit toro))
+  %+  poke-info  "removed: {<a>}"
+  =-  %+  roll  -
+  |=  [a=(unit toro) b=(unit toro)]
```
Comment (belisarius222) [inline pkg/arvo/lib/hood/kiln.hoon:1262]: This line and the next should be indented twice to match the `%+  roll`

**Comment by zalberico** [comment]
@ryjm can you clarify what this is actually for or write up and link it to an issue about what problem this is solving? As is it's hard for someone to evaluate and approve/merge.

**Comment by belisarius222** [comment]
|rm removes files from Clay, just like Unix "rm". It's a cleaner interface if it's recursive, in the sense that it deletes all files whose paths start with the supplied path (like "rm -r"), but it wasn't recursive until this PR.

—
Ted Blackman
CTO | Urbit Foundation
~rovnys-ricfer
https://twitter.com/rovnys

**Comment by zalberico** [comment]
@belisarius222 thanks - is this approved/good to merge into next/arvo? We shouldn't be merging straight into master outside of releases.

**Comment by belisarius222** [comment]
Not ready for merge, still needs to handle the case where fil and dir are both non-null (I just talked to Jake about this). Should also target next/arvo, not master

—
Ted Blackman
CTO | Urbit Foundation
~rovnys-ricfer
https://twitter.com/rovnys

**Comment by zalberico** [comment]
@belisarius222 looks like @ryjm made the changes here, can you take a look?

**Review by philipcmonk — APPROVED** [review]
This works for me

**Pull request merged**